crates/syntax-bridge/src/prettify_macro_expansion.rs

```diff
@@ -8,10 +8,18 @@ use syntax::{
 };
 
 /// Renders a [`SyntaxNode`] with whitespace inserted between tokens that require them.
-pub fn insert_ws_into(syn: SyntaxNode) -> SyntaxNode {
+///
+/// This is an internal API that is only exported because `mbe` needs it for tests and cannot depend
+/// on `hir-expand`. For any purpose other than tests, you are supposed to use the `prettify_macro_expansion`
+/// from `hir-expand` that handles `$crate` for you.
+pub fn prettify_macro_expansion(
```

Comment:
A trick we use for other similar "don't use" APIs is putting a `#[deprecrated = "msg"]` on it and have it allowed on the few callers